src/test/regress/expected/isolation_append_copy_vs_all.out

```diff
@@ -102,7 +102,7 @@ step s1-commit: COMMIT;
 step s1-select-count: SELECT COUNT(*) FROM append_copy;
 count
 
-10
+15
```

Comment:
This change is quite strange. 15 looks like the correct result to me, but I need to make sure.

@onderkalaci do you think this change is expected here?